config/routes.rb

```diff
@@ -60,7 +60,13 @@
   resource :school, only: [] do
     get 'customize'
     get 'admins'
+    get 'standing'
```

Comment:
I thought the other way around: 😅

Actually, that page does more than just listing all the available standings in a school.

- It also includes an option to enable and disable the standing 
- Link to edit and view the COC

the singular naming –69 standing – is intentional. It reflects the action's role in managing the overall "Standing" feature as a holistic entity under school, rather than just representing an index of individual standings.

I would also like to revert the `Standings` name on the left secondary nav to `Standing`🤔

---

I will correct the line (69)